lib/improver/utilities/mathematical_operations.py

```diff
@@ -209,9 +237,44 @@ def process(self, cube):
             stride_sum += lower_half_of_stride + upper_half_of_stride
             integrated_slice.data = stride_sum
             integrated_cubelist.append(integrated_slice.copy())
-
         # Merge resulting cubes back together
         integrated_cube = integrated_cubelist.merge_cube()
+        return integrated_cube
+
+    def process(self, cube):
+        """Integrate a specified coordinate. This is calculated by defining the
+        upper and lower bounds for the steps along a chosen coordinate
+        within the cube.
+
+        Functions utilised are:
+            1. Ensure the cube is sorted in the direction desired for
+               integration.
+            2. Prepare for integration by creating cubes that represent the
+               upper and lower limits of the integration, as well as as a
+               template cube to put the integrated output.
+            3. Perform the integration using the trapezoidal rule.
+            4. Ensure that the integrated coordinate is a dimension coordinate
+               and ensure that the integrated coordinate is sorted in the
+               desired direction.
+
+        Args:
+            cube (Iris.cube.Cube):
+                Cube containing the data to be integrated.
```

Comment:
I think we should state here that cube could be modified by this function - the ensure_monotonic function will alter it in-place if necessary.

Reply:
Added a note.